templates/td-agent-bit.conf.j2

```diff
@@ -65,6 +65,13 @@
 {% endif %}
 {% endfor %}
 
+{% for _postfilter in _fluentbit_postfilters %}
+[FILTER]
```

Comment:
maybe add support for list of dict (is mapping) ?

Reply:
Added and tested